test/replacer.js

```diff
@@ -31,6 +31,6 @@ describe('replacer option', function() {
       return k;
     };
     
-    hash.writeToStream({foo: 'bar'}, {excludeValues: true}, strm);
+    hash.writeToStream({foo: 'bar', replacer: replacer}, {excludeValues: true}, strm);
```

Comment:
This is a fix for the tests. The `replacer` variable was not used, which was previously not detected, as JSHint wasn't running on the test files.